pkg/dashboard/templates/container.gohtml

```diff
@@ -111,6 +116,11 @@
     <a href="#burstable-qos"
       >Burstable <abbr title="Quality of Service">QoS</abbr></a
     >
+    {{if lt $.BurstableCostInt 0}}
+      <span class="lower-number lower-number--negative">-${{$.BurstableCost}}/hour</span>
+    {{else if gt $.BurstableCostInt 0}}
```

Comment:
These aren't showing up for me. Maybe this should just be an `else` instead of an `else if`?

Reply:
I updated the code. Thank you